lldb/tools/lldb-dap/JSONUtils.h

```diff
@@ -437,7 +434,7 @@ std::string CreateUniqueVariableNameForDisplay(lldb::SBValue v,
 ///     context (e.g. locals). This can happen due to shadowed variables in
 ///     nested blocks.
 ///
-///     As VSCode doesn't render two of more variables with the same name, we
```

Comment:
Same here, "VSCode doesn't render..." looks like it's talking about VSCode the program rather than the protocol, and how we workaround this VSCode-specific behavior.